draft-ietf-rats-eat.md

```diff
@@ -288,54 +288,40 @@ An entity may have strong security defenses against hardware invasive attacks.
 It may also have low security, having no special security defenses.
 There is no minimum security requirement to be an entity.
 
-## CWT, JWT and Detached EAT Bundle
 
-An EAT is primarily a claims set about an entity based on one of the following:
+## EAT as a Framework
 
-* CBOR Web Token (CWT) {{RFC8392}}
-* JSON Web Token (JWT) {{RFC7519}}
+EAT can be viewed as a framework for defining attestation tokens for specific use cases more so than it is one specific token definition.
+While EAT is based on and compatible with CWT and JWT, it can be thought of in these terms:
```

Comment:
Maybe keep it crisp here, too? Such as:
```suggestion
EAT provides the following features:
```

Reply:
Yes. Made a separate change in this spirit.